storage/src/state/tests.rs

```diff
@@ -32,6 +32,14 @@ fn create_new_ledger<N: Network, S: Storage>() -> LedgerState<N> {
     LedgerState::open_writer_with_increment::<S, _>(temp_dir(), 1).expect("Failed to initialize ledger")
 }
 
+fn to_hash_set_test<N: Network>(ciphertexts: Vec<N::RecordCiphertext>) -> HashSet<N::RecordCiphertext> {
+    let mut set = HashSet::new();
+    for ciphertext in ciphertexts {
+        set.insert(ciphertext);
+    }
+    set
+}
```

Comment:
I don't think this helper is needed, this is just `ciphertexts.into_iter().collect::<HashSet<_>>()`

Reply:
You're right. I deleted this helper function and change the collect type on the last commit.